modules/sheet/editTags.js

```diff
@@ -0,0 +1,183 @@
+const nest = require('depnest')
+const { h, Value, Struct, map, computed } = require('mutant')
+const MutantArray = require('mutant/Array')
+const concat = require('lodash/concat')
+const filter = require('lodash/filter')
+const zip = require('lodash/zip')
+const forEach = require('lodash/forEach')
+const addSuggest = require('suggest-box')
+
+exports.gives = nest('sheet.editTags')
+
+exports.needs = nest({
+  'about.obs.name': 'first',
+  'keys.sync.id': 'first',
+  'sheet.display': 'first',
+  'tag': {
+    'async': {
+      'apply': 'first',
+      'create': 'first',
+      'name': 'first'
+    },
+    'html': {
+      'tag': 'first'
+    },
+    'obs': {
+      'messageTagsFrom': 'first',
+      'allTags': 'first'
```

Comment:
Matt might have a different preference but I prefer all depject things to be flat to make searching the project for uses easier. 
i.e. `tag.obs.messageTagsFrom` instead of that nested thing.

I haven't looked at the rest of the project recently to gauge style to be fair